app/app/controllers/caseworker/cbv_flow_invitations_controller.rb

```diff
@@ -72,7 +86,8 @@ def cbv_flow_invitation_params
       :email_address,
       :snap_application_date,
       :agency_id_number,
-      :beacon_id
+      :beacon_id,
+      :welid
```

Comment:
The field is still named `beacon_id` even though it is labeled in the UI as the "WELID", right?

```suggestion
      :beacon_id
```